### ISSUE TITLE
Redirection not working after setUser()

### ISSUE THREAD

**Issue by Panksi** [nuxt-community/auth-module#1106]
I am using local strategy:
![image](https://user-images.githubusercontent.com/16855132/113207846-68137f00-9271-11eb-99a6-eb9701706e46.png)

and I am getting user data in login request.
So I used $auth.setUser() method to set the data.
![image](https://user-images.githubusercontent.com/16855132/113207800-5c27bd00-9271-11eb-9160-11f132450f7b.png)

The issue is that I am not getting redirected even though I have all the data in my $auth state.
$state:
busy: false
loggedIn: true
redirect: "/"
strategy: "local"
user:
firstName: "Scott"
lastName: "Buell"

**Comment by IsidroMar95** [comment]
same issue here, the login works but not redirect to home page.

**Comment by productfrontenddeveloper** [comment]
your write custom axios settings ?

**Comment by Panksi** [comment]
> your write custom axios settings ?

not really. I have used extended local scheme and $auth setUser method, as per documentation.

**Comment by Panksi** [comment]
@IsidroMar95 I fixed this with a workaround. I used universal storage for it. Find it in documentation https://auth.nuxtjs.org/api/storage/

**Comment by rod-dd** [comment]
Ive encountered also the similar problem here, after using the .setUser in page1 then access the user on page2, i cant access the set data. so with your solution @Panksi  i managed to get over on this. thanks.
